pkgs/tools/security/pomerium-cli/default.nix

```diff
@@ -9,15 +9,17 @@ let
 in
 buildGoModule rec {
   pname = "pomerium-cli";
-  version = pomerium.version;
+  inherit (pomerium) version;
+
   src = fetchFromGitHub {
     owner = "pomerium";
     repo = "cli";
     rev = "v${version}";
-    hash = "sha256:0230b22xjnpykj8bcdahzzlsvlrd63z2cmg6yb246c5ngjs835q1";
```

Comment:
I don't mind you changing this, but bear in mind this is likely to backslide if I update this package since this is still the hash format I get out of the Nix CLI.

Reply:
How? The `nix` standalone tool uses SRI, indeed the command `nix hash to-sri` can be employed to convert them.

Also, the format with `<algo>:<hash>` is being discouraged anyway.